node.go

```diff
@@ -69,9 +87,20 @@ func (node *node) getChannel() (*channel, error) {
 		PermitWithoutStream: true,
 	}
 
-	conn, err := grpc.Dial(node.address, grpc.WithInsecure(), grpc.WithKeepaliveParams(kacp), grpc.WithBlock())
+	var conn *grpc.ClientConn
+	var err error
+	parts := strings.SplitN(node.address, ":", 2)
+	if parts[1] == "443" || parts[1] == "50212" {
+		cont, cancel := context.WithTimeout(context.Background(), 10*time.Second)
+		defer cancel()
+		conn, err = grpc.DialContext(cont, node.address, grpc.WithTransportCredentials(credentials.NewTLS(&tls.Config{InsecureSkipVerify: true})), grpc.WithKeepaliveParams(kacp), grpc.WithBlock())
```

Comment:
Instead of making a conditional and calling `grpc.DialContext()` context you can move that to outside the conditional and instead only save the security inside the conditional.
```go
security := grpc.WithInsecure()

if parts[1] == "443" || parts[1] == "50212" {
    security = grpc.WithTransportCredentials(credentials.NewTLS(&tls.Config{InsecureSkipVerify: true}))
}

conn, err := grpc.Dial(cont, node.address, security, grpc.WithKeepaliveParams(kacp), grpc.WithBlock())
if err != nil {
    panic(err)
}
```